server/models/votes.model.js

```diff
@@ -0,0 +1,55 @@
+import { Model } from 'objection';
+
+export default class Vote extends Model {
+  static get tableName() {
```

Comment:
nit: `static tableName = ‘votes’;`

Reply:
tableName is a feathers-objection supported method